File: revolut/Account.m
//
//  Account.m
//  revolut
//
//  Created by Alexander Danilov on 02/09/2017.
//  Copyright © 2017 Home. All rights reserved.
//

#import "Account.h"

@implementation Account
    
@synthesize currency = _currency;
@synthesize balance = _balance;
    
-(instancetype) initWithCurrency: (NSString *)currency andBalance:(NSDecimalNumber *)balance {
    self = [super init];
    if (self) {
        _currency = currency;
        _balance = balance;
    }
    return self;
}

@end

File: revolut/Account.swift
//
//  Account.swift
//  revolut
//
//  Created by Alexander Danilov on 12/08/2018.
//  Copyright © 2018 Home. All rights reserved.
//

import Foundation

class Account: NSObject {
    var currency: String
    var balance: NSDecimalNumber
    
    init(currency: String, balance: NSDecimalNumber) {
        self.currency = currency
        self.balance = balance
    }
}
